hazelcast/src/main/resources/hazelcast-config-5.3.json

```diff
@@ -236,6 +236,9 @@
         },
         "sql": {
           "$ref": "#/definitions/ClientSql"
+        },
+        "alto": {
```

Comment:
This comment is general about this file. WDYT about adding extra item to https://github.com/hazelcast/hazelcast/wiki/Checklist-for-Hazelcast-config-changes#hazelcast-client-config because this json file is listed under member config right now but client config is also included in this file.

Reply:
OK, I have added one item to the client section, although we sort of describe the process in the separate section for the client as well
```
hazelcast-config-x.y.json for the client-side changes, as described in the above section
```